.travis.yml

```diff
@@ -3,7 +3,6 @@ language: node_js
 node_js:
   - "0.10.40"
 install:
-  - "npm install -g eslint"
-  - "npm install -g eslint-plugin-meteor"
+  - "npm install"
 script:
-  - "eslint ./"
+  - "npm test"
```

Comment:
I think this doesn’t work as intented because even if some tests fails the unix return code will be `0` instead of a greater number as it does with `eslint ./` or even `npm run lint`. Could you verify @dferber90?